src/libambit/device_driver_ambit3.c

```diff
@@ -579,7 +579,10 @@ static int parse_log_header(libambit_sbem0102_data_t *reply_data_object, ambit3_
         data += 19;
         break;
       case 0x8a: /* gen2+gen3 fw */
-        data += 1; // Skip size ??
+        if (fw_gen == AMBIT3_FW_GEN3)
+            data += 1; // Skip size
```

Comment:
Thanks for your usb dump. I have to look more into this log, because there is a difference here.

I now understand why it works on your ambit.